pkg/agent/sysadvisor/plugin/qosaware/resource/cpu/region/headroompolicy/policy_canonical.go

```diff
@@ -17,16 +17,14 @@ limitations under the License.
 package headroompolicy
```

Comment:
if this policy is deprecated, which one should we use? will there be a corresponding rama headroom policy?

Reply:
There may be a policy based on machine learning to estimate NUMA headroom for dedicated numa exclusive region in the future.